### ISSUE TITLE
Need Update 2.1.7 please

### ISSUE THREAD

**Issue by Haecky** [tystuyfzand/gotify-smtp#6]
Need Update 2.1.7 please

**Comment by TheLastProject** [comment]
First of all I want to say: thank you tystuyfzand for this plugin. I use it to get Nextcloud notifications in Gotify and it always served me well.

Having an update would be great, Using the current plugin version with Gotify 2.1.6 or 2.1.7 causes Gotify to crash on start:

<details><summary>Crash log</summary><br>

```
Starting Gotify version 2.1.7@2022-09-11-07:29:06
fatal error: runtime: no plugin module data

goroutine 1 [running]:
runtime.throw({0x1054c57?, 0x739bfb?})
	/usr/local/go/src/runtime/panic.go:1047 +0x5d fp=0xc0004bd440 sp=0xc0004bd410 pc=0x70c93d
plugin.lastmoduleinit()
	/usr/local/go/src/runtime/plugin.go:20 +0x8ce fp=0xc0004bd528 sp=0xc0004bd440 pc=0x739a4e
plugin.open({0xc0000b0720, 0x20})
	/usr/local/go/src/plugin/plugin_dlopen.go:77 +0x3ef fp=0xc0004bd798 sp=0xc0004bd528 pc=0xd601ef
plugin.Open(...)
	/usr/local/go/src/plugin/plugin.go:32
github.com/gotify/server/v2/plugin.(*Manager).loadPlugins(0x203000?, {0xc0003ff9b0, 0xc})
	/proj/plugin/manager.go:230 +0x190 fp=0xc0004bd890 sp=0xc0004bd798 pc=0xd63c70
github.com/gotify/server/v2/plugin.NewManager({0x1543960?, 0xc0000143b8}, {0xc0003ff9b0, 0xc}, 0xc0004e67c0, {0x1539d00?, 0xc0004e6780})
	/proj/plugin/manager.go:86 +0x26f fp=0xc0004bd9e0 sp=0xc0004bd890 pc=0xd62aef
github.com/gotify/server/v2/router.Create(0xc0000143b8, 0xc000016ff0, 0xc0000f2300)
	/proj/router/router.go:46 +0x827 fp=0xc0004bdeb0 sp=0xc0004bd9e0 pc=0xdea0c7
main.main()
	/proj/app.go:51 +0x307 fp=0xc0004bdf80 sp=0xc0004bdeb0 pc=0xe0ac67
runtime.main()
	/usr/local/go/src/runtime/proc.go:250 +0x212 fp=0xc0004bdfe0 sp=0xc0004bdf80 pc=0x70f4b2
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc0004bdfe8 sp=0xc0004bdfe0 pc=0x73e901

goroutine 2 [force gc (idle)]:
runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
	/usr/local/go/src/runtime/proc.go:363 +0xd6 fp=0xc000054fb0 sp=0xc000054f90 pc=0x70f876
runtime.goparkunlock(...)
	/usr/local/go/src/runtime/proc.go:369
runtime.forcegchelper()
	/usr/local/go/src/runtime/proc.go:302 +0xad fp=0xc000054fe0 sp=0xc000054fb0 pc=0x70f70d
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc000054fe8 sp=0xc000054fe0 pc=0x73e901
created by runtime.init.6
	/usr/local/go/src/runtime/proc.go:290 +0x25

goroutine 3 [GC sweep wait]:
runtime.gopark(0x1?, 0x0?, 0x0?, 0x0?, 0x0?)
	/usr/local/go/src/runtime/proc.go:363 +0xd6 fp=0xc000055790 sp=0xc000055770 pc=0x70f876
runtime.goparkunlock(...)
	/usr/local/go/src/runtime/proc.go:369
runtime.bgsweep(0x0?)
	/usr/local/go/src/runtime/mgcsweep.go:297 +0xd7 fp=0xc0000557c8 sp=0xc000055790 pc=0x6fa6d7
runtime.gcenable.func1()
	/usr/local/go/src/runtime/mgc.go:178 +0x26 fp=0xc0000557e0 sp=0xc0000557c8 pc=0x6ef266
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc0000557e8 sp=0xc0000557e0 pc=0x73e901
created by runtime.gcenable
	/usr/local/go/src/runtime/mgc.go:178 +0x6b

goroutine 4 [GC scavenge wait]:
runtime.gopark(0xc00007a000?, 0x15361a8?, 0x0?, 0x0?, 0x0?)
	/usr/local/go/src/runtime/proc.go:363 +0xd6 fp=0xc000055f70 sp=0xc000055f50 pc=0x70f876
runtime.goparkunlock(...)
	/usr/local/go/src/runtime/proc.go:369
runtime.(*scavengerState).park(0x1a4d320)
	/usr/local/go/src/runtime/mgcscavenge.go:389 +0x53 fp=0xc000055fa0 sp=0xc000055f70 pc=0x6f86b3
runtime.bgscavenge(0x0?)
	/usr/local/go/src/runtime/mgcscavenge.go:622 +0x65 fp=0xc000055fc8 sp=0xc000055fa0 pc=0x6f8cc5
runtime.gcenable.func2()
	/usr/local/go/src/runtime/mgc.go:179 +0x26 fp=0xc000055fe0 sp=0xc000055fc8 pc=0x6ef206
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc000055fe8 sp=0xc000055fe0 pc=0x73e901
created by runtime.gcenable
	/usr/local/go/src/runtime/mgc.go:179 +0xaa

goroutine 5 [finalizer wait]:
runtime.gopark(0x1a4e1e0?, 0xc000007860?, 0x0?, 0x0?, 0xc000054770?)
	/usr/local/go/src/runtime/proc.go:363 +0xd6 fp=0xc000054628 sp=0xc000054608 pc=0x70f876
runtime.goparkunlock(...)
	/usr/local/go/src/runtime/proc.go:369
runtime.runfinq()
	/usr/local/go/src/runtime/mfinal.go:180 +0x10f fp=0xc0000547e0 sp=0xc000054628 pc=0x6ee36f
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc0000547e8 sp=0xc0000547e0 pc=0x73e901
created by runtime.createfing
	/usr/local/go/src/runtime/mfinal.go:157 +0x45

goroutine 34 [GC worker (idle)]:
runtime.gopark(0x14?, 0x160?, 0xa0?, 0x67?, 0x6db239?)
	/usr/local/go/src/runtime/proc.go:363 +0xd6 fp=0xc000056750 sp=0xc000056730 pc=0x70f876
runtime.gcBgMarkWorker()
	/usr/local/go/src/runtime/mgc.go:1235 +0xf1 fp=0xc0000567e0 sp=0xc000056750 pc=0x6f13b1
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc0000567e8 sp=0xc0000567e0 pc=0x73e901
created by runtime.gcBgMarkStartWorkers
	/usr/local/go/src/runtime/mgc.go:1159 +0x25

goroutine 18 [GC worker (idle)]:
runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
	/usr/local/go/src/runtime/proc.go:363 +0xd6 fp=0xc000050750 sp=0xc000050730 pc=0x70f876
runtime.gcBgMarkWorker()
	/usr/local/go/src/runtime/mgc.go:1235 +0xf1 fp=0xc0000507e0 sp=0xc000050750 pc=0x6f13b1
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc0000507e8 sp=0xc0000507e0 pc=0x73e901
created by runtime.gcBgMarkStartWorkers
	/usr/local/go/src/runtime/mgc.go:1159 +0x25

goroutine 7 [GC worker (idle)]:
runtime.gopark(0x43fa4ccfcd511?, 0x0?, 0x0?, 0x0?, 0x0?)
	/usr/local/go/src/runtime/proc.go:363 +0xd6 fp=0xc000056f50 sp=0xc000056f30 pc=0x70f876
runtime.gcBgMarkWorker()
	/usr/local/go/src/runtime/mgc.go:1235 +0xf1 fp=0xc000056fe0 sp=0xc000056f50 pc=0x6f13b1
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc000056fe8 sp=0xc000056fe0 pc=0x73e901
created by runtime.gcBgMarkStartWorkers
	/usr/local/go/src/runtime/mgc.go:1159 +0x25

goroutine 19 [GC worker (idle)]:
runtime.gopark(0x43fa4ccfde73d?, 0x0?, 0x0?, 0x0?, 0x0?)
	/usr/local/go/src/runtime/proc.go:363 +0xd6 fp=0xc000050f50 sp=0xc000050f30 pc=0x70f876
runtime.gcBgMarkWorker()
	/usr/local/go/src/runtime/mgc.go:1235 +0xf1 fp=0xc000050fe0 sp=0xc000050f50 pc=0x6f13b1
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc000050fe8 sp=0xc000050fe0 pc=0x73e901
created by runtime.gcBgMarkStartWorkers
	/usr/local/go/src/runtime/mgc.go:1159 +0x25

goroutine 8 [select]:
runtime.gopark(0xc000051788?, 0x2?, 0x0?, 0x0?, 0xc000051784?)
	/usr/local/go/src/runtime/proc.go:363 +0xd6 fp=0xc000051610 sp=0xc0000515f0 pc=0x70f876
runtime.selectgo(0xc000051788, 0xc000051780, 0x0?, 0x0, 0x0?, 0x1)
	/usr/local/go/src/runtime/select.go:328 +0x7bc fp=0xc000051750 sp=0xc000051610 pc=0x71ee7c
database/sql.(*DB).connectionOpener(0xc000415ee0, {0x1540158, 0xc0003cacc0})
	/usr/local/go/src/database/sql/sql.go:1224 +0x8d fp=0xc0000517b8 sp=0xc000051750 pc=0xc995cd
database/sql.OpenDB.func1()
	/usr/local/go/src/database/sql/sql.go:792 +0x2e fp=0xc0000517e0 sp=0xc0000517b8 pc=0xc9788e
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc0000517e8 sp=0xc0000517e0 pc=0x73e901
created by database/sql.OpenDB
	/usr/local/go/src/database/sql/sql.go:792 +0x18d

goroutine 35 [chan receive]:
runtime.gopark(0x0?, 0xc0002d26e8?, 0xdf?, 0x22?, 0x0?)
	/usr/local/go/src/runtime/proc.go:363 +0xd6 fp=0xc0002d2668 sp=0xc0002d2648 pc=0x70f876
runtime.chanrecv(0xc0004fe1e0, 0xc0002d2770, 0x1)
	/usr/local/go/src/runtime/chan.go:583 +0x49b fp=0xc0002d26f8 sp=0xc0002d2668 pc=0x6db7fb
runtime.chanrecv1(0x0?, 0x0?)
	/usr/local/go/src/runtime/chan.go:442 +0x18 fp=0xc0002d2720 sp=0xc0002d26f8 pc=0x6db2f8
github.com/gotify/server/v2/plugin.NewManager.func1()
	/proj/plugin/manager.go:69 +0xd2 fp=0xc0002d27e0 sp=0xc0002d2720 pc=0xd62d32
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc0002d27e8 sp=0xc0002d27e0 pc=0x73e901
created by github.com/gotify/server/v2/plugin.NewManager
	/proj/plugin/manager.go:67 +0x252
```
</details>

Thank you in advance for looking into this :)

**Comment by tystuyfzand** [comment]
Looks like Huginn failed to trigger the job to auto build once Gotify released 2.1.6 and 2.1.7, they should be running now!

Edit: It's now built: https://github.com/tystuyfzand/gotify-smtp/releases/tag/latest-gotify-v2.1.7
